amp/TEMPLATE/reampv2/src/modules/sscdashboard/components/layout/filters/horizontal-filters.jsx

```diff
@@ -39,7 +40,15 @@ class HorizontalFilters extends Component {
         });
     }
 
+    clearFilters(event) {
+        this.props.handleSelectedCountryChanged([]);
```

Comment:
Why 2 times?

Reply:
the problem was worse, I change the handler sent to the sector filter. Thanks for spotting this